cmd/pulumi-test-language/interface.go

```diff
@@ -830,6 +847,11 @@ func (eng *languageTestServer) RunLanguageTest(
 			if err != nil {
 				return nil, err
 			}
+			if pkg == "parameterized" {
+				// TODO: not sure what to do with this yet because we can't get the parameterized package name
+				// unless we call `Parameterize` which we can't do here because we don't have the descriptor info
+				continue
+			}
```

Comment:
So I think we are going to have to refactor conformance here so that instead of a list of providers per test, we provide a list of packages, to solve precisely this problem.

Reply:
Add issue about extending providers definition include parameterization